doc/nrf/releases_and_maturity/migration/migration_guide_2.7.rst

```diff
@@ -125,6 +125,14 @@ Matter
       * All Partition Manager configuration files (:file:`pm_static` files) have been removed from the :file:`configuration` directory.
         Instead, a :file:`pm_static_<BOARD>` file has been created for each target board and placed in the samples' directories.
         Setting the ``PM_STATIC_YML_FILE`` argument in the :file:`CMakeLists.txt` file has been removed, as it is no longer needed.
+      * Configuration files :file:`Kconfig.mcuboot.defaults`, :file:`Kconfig.hci_ipc.defaults` and :file:`Kconfig.multiprotocol_rpmsg.defaults` that stored a default configuration for the child images have been removed.
+        This was done because of the sysbuild integration and child images deprecation.
+        The Matter samples and applications have been migrated to use sysbuild, however you can still use the child images.
```

Comment:
```suggestion
        This was done because of the sysbuild integration and child images deprecation.

        The Matter samples and applications have been migrated to use sysbuild, though you can still use the child images.
```